libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParserListener.ts

```diff
@@ -85,17 +98,17 @@ export interface ExpressionAntlrParserListener extends ParseTreeListener {
 	exitStringAtom?: (ctx: StringAtomContext) => void;
 
 	/**
-	 * Enter a parse tree produced by the `indexAccessExp`
+	 * Enter a parse tree produced by the `idAtom`
 	 * labeled alternative in `ExpressionAntlrParser.primaryExpression`.
 	 * @param ctx the parse tree
 	 */
-	enterIndexAccessExp?: (ctx: IndexAccessExpContext) => void;
+	enterIdAtom?: (ctx: IdAtomContext) => void;
```

Comment:
This looks like a breaking change as [`IdAtomContext`](https://github.com/microsoft/botbuilder-js/blob/208f0fc7a4c535f7861ab95e9683a70b6d118545/libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParser.ts#L1322) does not have the same signature of [`IndexAccessExpContext`](https://github.com/microsoft/botbuilder-js/blob/8c15615dd5cf6dc6817a0771cb7ea3bf85e07455/libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParser.ts#L1295).

Reply:
there is no breaking change on parser, it is just the misconception caused by the dislocation of the code diff.